turbopuffer/backend.py

```diff
@@ -71,16 +79,40 @@ def make_api_request(self,
         prepared = self.session.prepare_request(request)
 
         retry_attempts = 0
-        while retry_attempts < 3:
-            # before = time.monotonic()
+        while retry_attempts < 6:
+            if retry_attempts > 0:
+                print("retrying...")
```

Comment:
I'm picturing how this would be used in a production application. Generally libraries are not expected to log at all, but pass telemetry/errors back to the application to decide what to do. Seeing a log line that just says "retrying..." in a soup of potentially many other lines emitted by an application (especially an async application) wouldn't be very helpful.

Reply:
I'll remove the retry logging for now, and set it up so it only logs the last error if the max retries was exceeded. Maybe a future version can do some fancier logging configuration.